sdks/python/apache_beam/runners/direct/util.py

```diff
@@ -26,10 +26,11 @@
 class TransformResult(object):
   """Result of evaluating an AppliedPTransform with a TransformEvaluator."""
 
-  def __init__(self, applied_ptransform, uncommitted_output_bundles,
+  def __init__(self, transform_evaluator, uncommitted_output_bundles,
                unprocessed_bundles, counters, keyed_watermark_holds,
-               undeclared_tag_values=None):
-    self.transform = applied_ptransform
+               undeclared_tag_values=None,
+               partial_keyed_state=None, keyed_existing_state=None):
+    self.transform = transform_evaluator._applied_ptransform # mgh
```

Comment:
<!--new_thread; commit:6ce5f121f4998fe51aa9ac13064dce692cd1228b; resolved:0-->
You can remove the comment here.